libs/cardano-ledger-api/src/Cardano/Ledger/Api/State/Query.hs

```diff
@@ -100,6 +101,7 @@ queryDRepState nes creds
 -- | Query DRep stake distribution. Note that this can be an expensive query because there
 -- is a chance that current distribution has not been fully computed yet.
 queryDRepStakeDistr ::
+  ConwayEraGov era =>
```

Comment:
Unfortunately we can't use `ConwayEraGov` constraint here.

All queries must work for all eras starting with Shelley, even if they return empty results for some of them.

Which I think can be solved by adding a function that gets the distribution to the `EraGov` type class: 
```haskell
  getDRepDistr :: GovState era -> Map (DRep (EraCrypto era)) (CompactForm Coin)
```

It would return an empty `Map` for all eras and in conway it could use the new `drepDistrGovStateL` lens to get the distr

```suggestion
  EraGov era =>
```